src/emissions.py

```diff
@@ -1063,6 +1052,76 @@ def calculate_weighted_nox_rates(year, nox_rates, aggregation_level):
     return weighted_nox_rates
 
 
+def associate_control_ids_with_boiler_id(df, year, pollutant):
+    """Associates emission control ids with boiler_id.
+
+    Because some emission control data is missing a corresponding control id for that pollutant,
+    this function attempts to use all control ids to associate each obervation with a boiler id,
+    in the order specified by `id_order`
+    """
+
+    all_pollutants = ["pm", "so2", "nox", "hg"]
```

Comment:
I assume that changing this order causes few or no differences in the resulting map between emission control IDs and boiler_id?

Reply:
That is correct - that is something I checked.